spec/integration/cli_verify_spec.rb

```diff
@@ -78,5 +79,22 @@
 EOT
       expect(result).to exit_with_error(1,"",expected_output)
     end
+
+    it "verifies manifests with topics warning" do
+      dir = given_directory do
+        given_directory_from_data("awesomelib", from: "manifests/awesomelib")
+        given_directory_from_data("missing-topics", from: "missing-topics")
+      end
+
+      result = run_command(args: ["verify", "--offline", "--manifest_dir=#{dir}"])
+      expected_output = <<EOT
+Verify manifest awesomelib.2013-09-08.manifest...ok
+Verify manifest missing-topics.manifest...ok
+  Warning: missing `topics` attribute
+
+2 manifests checked. 1 ok, 0 with error, 1 warning.
```

Comment:
To make it more clear maybe the message should say "1 have warnings", so that it's clear that this is the number of manifests which have warnings, not the number of warnings in total.